### PR TITLE
Minor tweaks around attributes, cyber and normal.

### PR DESCRIPTION
**Pull request by oddmage**
Added some stuff, I should have lots of time this week and next, so if there's something you'd like me to try and hit up, let me know.

### CODE REVIEW

File: src/components/gear/cyberware/CyberlimbAttributeComponent.js [hunk 1]
```diff
@@ -1,26 +1,31 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import ModificationButton from '../../ModificationButton';
 
-const CyberlimbAttribute = ({incrementAttribute, decrementAttribute, attribute, att}) => {
+const CyberlimbAttribute = ({incrementAttribute, decrementAttribute, attributeValue, attributeName}) => {
```
Comment (dethstrobe) [inline src/components/gear/cyberware/CyberlimbAttributeComponent.js]: Way more logical naming convention. Don't know why I didn't think of it myself.

File: src/components/ModificationButton.js [hunk 2]
```diff
@@ -3,37 +3,38 @@ import PropTypes from 'prop-types';
...
 	modificationFunction: PropTypes.func.isRequired,
-	attType: PropTypes.string.isRequired,
+	attType: PropTypes.string,
 	symbol: PropTypes.string.isRequired
 };
 
 ModificationButton.defaultProps = {
-	pointsLeft: 1
+	pointsLeft: 1,
+	attName: '!!!!!!!!!!!!!!!!!!!!',
```
Comment (dethstrobe) [inline src/components/ModificationButton.js]: Aren't `attName`, `maxPoints`, and `attType` all required? I can't think of a time we wouldn't have them defined.

**Comment by dethstrobe** [comment]
Good work, thanks a lot for your help.

If you want to take on something really big, you can try to work on building lifestyles.

But if you want to try and tackle something...small-ish-er... Being able to modify gear, like commlinks, cyberdecks, vision and audio devices, etc also still needs to get done. But I think with the way I've started to work on cyberlimb modding, it might be able to be translated into general gear too. But that's also a big maybe...because I don't know how much overlap there will actually be.

File: src/components/ModificationButton.js (hunk 2 above)
Comment (oddmage) [reply src/components/ModificationButton.js]: You don't need them if you pass in a function that knows it already, is a subtracting button, and/or isn't limited by type.

File: src/components/gear/cyberware/CyberlimbAttributeComponent.js (hunk 1 above)
Comment (oddmage) [reply src/components/gear/cyberware/CyberlimbAttributeComponent.js]: Naming things well the first time is probably one of the hardest things to do.  Also switching between stuff, I kept losing track of what they meant, so I did it for my forgetful brain.

**Comment by oddmage** [comment]
Thanks and no problem.

For lifestyles it's just cost and label for the basic kind right? Or you want to do custom from the start?  I started working on the adept powers components, just pulling them into dumber components but could use more polishing.  Hopefully I'll be happy today at some point with it

**Comment by dethstrobe** [comment]
That sounds great. Adept powers are all kinds of messed up.

I have no idea how we want to handle lifestyles yet. Their rules are pretty complex.

**Comment by oddmage** [comment]
Could just do basic and then do add-ons

**Comment by dethstrobe** [comment]
That definitely makes the most sense.

**Pull request merged**